Core1Platform/HelpAndSupport/02_CreateTicket.md

```diff
@@ -0,0 +1,51 @@
+# Create ticket
+
+The creation of a ticket is necessary, if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   
+For example, one of the following situations may be a reason to create a ticket:   
+- A workflow is failing
+- The main account is not reachable
+- Products show incorrect information on the frontend   
+
+> [INFO] Before creating a ticket, it may be useful to first check whether another users have not already reported the discovered issue. To do this, you can check the *Consulting* (MPS agreement (Managed Professional Services)) or the *Support* (older agreement) module.
+
+
+
+## Create ticket
+
+Create a ticket for an issue that you have discovered. To speed up the support process we ask you to consider the instructions below when creating tickets.
+
+#### Prerequisites
+
+- You have the permission to create tickets.
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/UsingCore1/Core1Ticket.png "[Core1 Platform]")   
+
+1. Click the ![Support](../../Assets/Icons/CreateTicket.png "[Support]") (Support) button.
+The *Contact us* window is displayed.
+
+   ![Contact us](../../Assets/Screenshots/Core1Platform/UsingCore1/Core1TicketTemplate.png "[Contact us")   
+
+2. Enter a descriptive reason in the *Reason* field. The reason description should be as precise as possible, for example "Delivery bills cannot be opened in the invoicing system".  An imprecise/bad example for the same fact would be "Invoicing does not work". 
+
+3. Enter a detailed description of the issue. Include the following information:
```

Comment:
3. und 4. sollten eigentlich in einem Schritt sein, da sie sich zum selben Field beziehen.